projects/core/bindaas-web-console/src/main/java/edu/emory/cci/bindaas/webconsole/PostLoginAction.java

```diff
@@ -119,5 +130,14 @@ private BindaasUser generateApiKey(BindaasUser principal) throws APIKeyManagerEx
 		principal.addProperty("apiKey", apiKey.getValue());
 		return principal;
 	}
+	private BindaasUser generateJWT(BindaasUser principal) throws JWTManagerException {
+
+		GregorianCalendar calendar = new GregorianCalendar();
+		calendar.add(Calendar.YEAR, 40);
+		Token token = JWTManager.generateJWT(principal, calendar.getTime(), "system", "System generated JWT for the user", ActivityType.SYSTEM_APPROVE, false);
+		log.info("Token for user: "+token.getValue());
+		principal.addProperty("apiKey",token.getValue());
```

Comment:
key used here is "apiKey" to make it work with the UI. Later commits have changed it to "jwt"